appveyor.yml

```diff
@@ -1,7 +1,7 @@
 version: 5.0.0.{build}
 clone_folder: c:\projects\nlogweb
 configuration: Release
-image: Visual Studio 2019
+image: Visual Studio 2022
```

Comment:
Thank you for bumping. Should bump the `image` for NLog and NLog.Extensions.Logging before NLog 5.0 is released.

Reply:
I had to for the code to compile against .NET 6 👍